lib/static/new-ui/utils/page.ts

```diff
@@ -0,0 +1,25 @@
+import {Page, PathNames} from '@/constants';
+
+export function getPageByPathname(pathname: string): Page {
+    if (pathname.startsWith(PathNames.visualChecks)) {
+        return Page.visualChecksPage;
+    }
+
+    if (pathname.startsWith(PathNames.suites)) {
+        return Page.suitesPage;
+    }
+
+    return Page.suitesPage;
```

Comment:
Redundant check: whats the point of "pathname.startsWith(PathNames.suites)", if we return "Page.suitesPage" in both cases?

I would write this function like this:
```js
return pathname.startsWith(PathNames.visualChecks) ? Page.visualChecksPage : Page.suitesPage;
```